opentech/apply/categories/blocks.py

```diff
@@ -63,7 +64,13 @@ def get_field_kwargs(self, struct_value):
 
     def get_widget(self, struct_value):
         if struct_value['multi']:
-            return forms.CheckboxSelectMultiple
+            category = struct_value['category']
+            category_size = len(category.options.values_list('id'))
+            # Pick widget according to number of options to maintain good usability.
+            if category_size < 32:
+                return forms.CheckboxSelectMultiple
+            else:
+                return Select2MultipleWidget()
```

Comment:
I would remove the call to the widget. I don't think we are currently passing arguments in when we are instantiating the objects, but it would be consistent with what the other widgets are doing.